tools/stats/stats.go

```diff
@@ -0,0 +1,32 @@
+package stats
+
+import (
+	"context"
+	"fmt"
+	influxdb2 "github.com/influxdata/influxdb-client-go"
+	"github.com/influxdata/influxdb-client-go/api"
+	"github.com/joschahenningsen/TUM-Live/model"
+	"time"
+)
+
+type Stats struct {
+	client    influxdb2.Client
+	liveStats api.WriteAPIBlocking
+}
+
+var Client *Stats
+
+func InitStats(client influxdb2.Client) {
+	Client = &Stats{
+		client:    client,
+		liveStats: client.WriteAPIBlocking("rbg", "live_stats"),
```

Comment:
This should either be non blocking or writing the stats has to happen in a separate go routine. I prefer the first option.